packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageScreen.tsx

```diff
@@ -22,15 +24,25 @@ const DynamicLeverageScreen = () => {
                 your trading position, based on asset type and trading volume.
             </WalletText>
             <div className='wallets-dynamic-leverage-screen__content'>
-                {dynamicLeverages.map(market => (
-                    <DynamicLeverageMarketCard
-                        data={market.data}
-                        description={market.description}
-                        key={`dynamic-leverage-screen__${market.key}`}
-                        leverage={market.leverage}
-                        title={market.title}
-                    />
-                ))}
+                {(['forex', 'metals', 'cryptocurrencies', 'stock_indices'] as const).map(key => {
```

Comment:
```suggestion
                {(['forex', 'metals', 'cryptocurrencies', 'stock_indices'] as const).map(key => {
```
cant you get these categories from the API?

Reply:
These categories are coming from API but i wanted them to be displayed in this order, according to the figma design.

Reply:
wew life as an FE is hard 😆